xray/audit/python/python.go

```diff
@@ -112,21 +112,15 @@ func runPythonInstall(tempDirPath string, pythonTool pythonutils.PythonTool) (re
 		}
 		// Run pip install
 		var output []byte
-		output, err = exec.Command("pip", "install", ".").CombinedOutput()
+		if requirementsFile != "" {
+			clientLog.Debug("Running pip install -r ", requirementsFile)
```

Comment:
```suggestion
			clientLog.Debug("Running pip install -r", requirementsFile)
```

Reply:
Added the space because otherwise, the log will be: "Running pip install -rrequirements.txt"